pkg/install/crdmigration/clones.go

```diff
@@ -863,6 +863,11 @@ func cloneEtcdRestoreResourcesInCluster(ctx context.Context, logger logrus.Field
 				BackupDownloadCredentialsSecret: oldObject.Spec.BackupDownloadCredentialsSecret,
 				BackupName:                      oldObject.Spec.BackupName,
 				Cluster:                         migrateObjectReference(oldObject.Spec.Cluster, ""),
+				Destination:                     oldObject.Spec.Destination,
+			},
+			Status: newv1.EtcdRestoreStatus{
+				Phase:       etcdRestoreStatusPhases[oldObject.Status.Phase],
+				RestoreTime: oldObject.Status.RestoreTime,
```

Comment:
No need to set the status here. It is done a few lines further down.

Reply:
well im blind, removing